.github/workflows/release.yaml

```diff
@@ -0,0 +1,182 @@
+#
+# Copyright Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+'
```

Comment:
Can we use a single pattern as in `.ci/cache_version_tags.sh` here as well?
`v[0-9]+.[0-9]+.[0-9]+(-rc.[0-9]+)?`

Reply:
GitHub workflow tag trigger is using a glob pattern instead of a regex pattern so the `(..)?` will not work here.